system/Test/bootstrap.php

```diff
@@ -49,6 +49,14 @@
 defined('COMPOSER_PATH') || define('COMPOSER_PATH', realpath(HOMEPATH . 'vendor/autoload.php'));
 defined('VENDORPATH')    || define('VENDORPATH', realpath(HOMEPATH . 'vendor') . DIRECTORY_SEPARATOR);
 
+/*
+ * ---------------------------------------------------------------
+ * CHECK PLATFORM REQUIREMENTS
+ * ---------------------------------------------------------------
+ */
+
+require SYSTEMPATH . 'platform_check.php';
```

Comment:
need to be before `new Paths()` to avoid class constant concat string error on php 5.5 ref https://3v4l.org/WvlTa#v5.5.30

Yes, I found a user ask in facebook why got error on upload to shared hostig